platform/linux-generic/odp_stash.c

```diff
@@ -432,22 +488,30 @@ int32_t odp_stash_get(odp_stash_t st, void *obj, int32_t num)
 	if (obj_size == sizeof(uint64_t)) {
 		ring_u64_t *ring_u64 = &stash->ring_u64.hdr;
 
-		return ring_u64_deq_multi(ring_u64, stash->ring_mask, obj, num);
+		if (batch)
```

Comment:
This could be done once at the beginning of the function to save correct functions to variables and then use those later.

Alternatively, correct functions could be passed directly to `stash_get()` (a bit clearer than passing a flag).

Reply:
Using function pointers would effectively prevent inlining. My target was to add support for batch operations with minimal overhead without having to duplicate the whole function.

Reply:
Sure, on the other hand, if inlined, it bloats the cache footprint as there are always the dead branches. I guess this would need to measured.

Anyways, current solution is OK to me as well.